tsconfig.json

```diff
@@ -0,0 +1,48 @@
+{
+  "compileOnSave": true,
```

Comment:
We can remove this. I added it unnecessarily to the `new-platform` and have forgotten about it.

Reply:
:heavy_check_mark: